pkgs/development/python-modules/terminaltables3/default.nix

```diff
@@ -0,0 +1,28 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  poetry-core,
+}:
+
+buildPythonPackage rec {
+  pname = "terminaltables3";
+  version = "4.0.0";
+  pyproject = true;
+
+  propagatedBuildInputs = [
+    poetry-core
+  ];
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-Tj7v4gmqiQBaCjTRUlc5QkVpcp7im15kqN1Rxevat38=";
+  };
+
```

Comment:
Could you enable tests with `pythonImportsCheck` and `pytestCheckHook`?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#testing-python-packages-testing-python-packages